packages/utils/src/common/object-to-string-path.ts

```diff
@@ -15,18 +15,25 @@ import { isObject } from "./is-object"
  *   test2: true
  * }
  * output: ['test.test1', 'test.test2', 'test.test3.test4', 'test2']
+ *
  * @param input
+ * @param includeTopLeaf
```

Comment:
question: The notion of a top leaf sounds off to me. Can you add a description + example to the JSDoc to make it more clear?

Reply:
Would `includeRootLevel` make more sense?

Reply:
Actually, this config seems to be very specific to the implementation. Is there a way we can name this prop, so it abstracts away the implementation specifics?

Reply:
I found it difficult to find the appropriate name so far

Reply:
@olivermrbl I tried to find an alternative naming and improve the example and description. Let me know what you think about it